packages/ckeditor5-typing/src/utils/inlinehighlight.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+import findAttributeRange from './findattributerange';
+
+/**
+ * @module typing/utils/inlinehighlight
+ */
+
+/**
+ * Adds a visual highlight style to an attribute element in which the selection is anchored.
```

Comment:
Not entirely true - there might be multiple attribute elements. But I don't have a better wording for this :/

Reply:
> Adds a visual highlight style to attribute elements bound to the anchored selection.

But "bound" is quite vague.

> Adds a visual highlight style to attribute elements adjacent to the anchored selection.

?

Reply:
Let's leave it as is. No need to debate on this ;)